appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 4.1.0.{build}
```

Comment:
@gerhardol This change should not have been part of this PR, right!?
We have to revert it?

Reply:
yes

Reply:
Due to rebasing 
I still prefer to cherry-pick from master to the release branches...